package.json

```diff
@@ -124,8 +125,9 @@
     "@types/pino": "^6.0.1",
     "@typescript-eslint/eslint-plugin": "^4.5.0",
     "@typescript-eslint/parser": "^4.5.0",
-    "ajv-errors": "^1.0.1",
-    "ajv-i18n": "^3.5.0",
+    "ajv": "^8.5.0",
```

Comment:
Why this is necessary?
I think the dependencies should not change for  this new parameter addition

Reply:
I was experimenting with bumping it because ajv doesn't get strict mode support until v7+, and otherwise there's no way for it to tell you that have shorthand schemas whos bare properties make it invalid. I was trying to add a test to get it to throw on receiving a shorthand schema with shorthand schemas disabled. Instead, without strict mode, it will just accept it as a catch-all.

Reply:
I've gone ahead and rolled this back, and just added to notes in the PR about the direction it will need to go for differentiating schemas.